src/audits/TableHasAppropriateHeaders.js

```diff
@@ -73,6 +73,32 @@ goog.require('axs.constants.Severity');
         }
         return false;
     }
+	
+    /**
+     * Checks whether a table is a layout table.
+     *
+     * @returns {boolean} Table is a layout table
+     */
+    function tableIsAFilthyLayoutTable(element) {
+        if (element.getAttribute('role') == 'presentation') {
+            return true;
+        }
+
+        if (element.hasAttribute("summary")) {
+            return true;
+        }
+			
+        // empty tables considered layout tables
+        if (!element.childElementCount > 0) {
```

Comment:
Also, note that if the only element in the table is a `<thead>` or `<tbody>` then the table will have child elements, but no actual cells. Perhaps do a querySelector for `tr` or `td`?